openstack/sharedfilesystems/v2/shares/requests.go

```diff
@@ -79,8 +79,11 @@ func Get(client *gophercloud.ServiceClient, id string) (r GetResult) {
 }
 
 // GetExportLocations will get shareID's export locations
```

Comment:
Would it be enough to improve the description of the GetExportLocations function? Would it be enough to document that client.Microversion must be set and that the value must be "2.14+"?

Note: discussion in https://github.com/gophercloud/gophercloud/issues/459 concluded that gophercloud shouldn't be responsible for unclear OpenStack errors.

Reply:
@pospispa ,Thanks for your comments.  Agree that it's strange that add microversion parameter into function since microversion is already a part of the client struct. So only improve the description of the function.